site/installing/standalone.md

```diff
@@ -3,4 +3,96 @@ title: Installing Weave Flux Standalone
 menu_order: 20
 ---
 
-_Content from deploy/standalone/README.md_
+Flux comes in several parts: 
+
+-   the command-line client **fluxctl**
+-   the service, which serves the API and
+-   the daemon, which carries out tasks on behalf of the service.
+
+Usually you would run the daemon yourself, and use the service that
+runs in Weave Cloud. But you can also run both of these in your own
+cluster.
+
+## Quick Install
```

Comment:
Ah, I see this is `standalone.md`, how about we just drop "Quick" from the title so folks don't get mislead by search engines etc?